conntrack_linux.go

```diff
@@ -384,7 +384,7 @@ func (f *ConntrackFilter) MatchConntrackFlow(flow *ConntrackFlow) bool {
 		match = match && elem.Equal(flow.Reverse.DstIP)
 	}
 
-	// -any-nat ip    Source or destination NAT ip
+	// Match source or destination reply IP
```

Comment:
As this comment is updated to match comment in consts also both two above comments could be also updated, but it's nothing blocking imo.